core/Makefile

```diff
@@ -110,14 +110,19 @@ pylint: ## run pylint on application sources and tests
 mypy:
 	mypy --config-file ../setup.cfg \
 		src/main.py \
-		src/apps/bitcoin \
-		src/apps/cardano \
-		src/apps/ethereum \
-		src/apps/management \
-		src/apps/misc \
-		src/apps/stellar \
-		src/apps/webauthn \
-		src/trezor/ui
+		src/apps \
```

Comment:
what is the reason for not specifying just `src/`?

Reply:
When specifying just `src`, it will "somehow" check the `monero` app as well, even when `--exclude` is there. Most probably some module in `src` is loading the monero app (and we could theoretically just exclude it as well)

Reply:
try excluding `all_modules.py`

Reply:
No luck, still it found the Monero app:
```
mypy --config-file ../setup.cfg \
    src \
    -exclude src/all_modules.py \
    --exclude src/apps/monero
```

Reply:
huh, looks like a mypy bug. hopefully it won't matter soon